File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/body/authentication/GetNewTokensBody.java
package com.lattis.ellipse.data.network.model.body.authentication;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 4/25/17.
 */

public class GetNewTokensBody {

    @SerializedName("user_id")
    public String userId;

    @SerializedName("password")
    public String password;

    public GetNewTokensBody(String userId, String password) {
        this.userId = userId;
        this.password = password;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/body/authentication/GetNewTokensBody.kt
package com.lattis.data.entity.body.authentication

import com.google.gson.annotations.SerializedName

/**
 * Created by ssd3 on 4/25/17.
 */

class GetNewTokensBody(@field:SerializedName("user_id")
                       var userId: String?, @field:SerializedName("password")
                       var password: String?)
